ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -731,6 +731,160 @@ task_definition:
 	}
 }
 
+func TestConvertToTaskDefinitionWithECSParams_ContainerResourcesPresent(t *testing.T) {
+	containerConfigs := testContainerConfigs([]string{"mysql", "wordpress"})
+
+	mysqlCPU := int64(100)
+	mysqlMem := int64(15)
+	mysqlMemRes := int64(10)
+
+	wordpressCPU := int64(4)
+	wordpressMem := int64(8)
+	wordpressMemRes := int64(5)
+
+	ecsParamsString := `version: 1
+task_definition:
+  services:
+    mysql:  
+      cpu_shares: 100
+      mem_limit: 15m
+      mem_reservation: 10m
+    wordpress:
+      cpu_shares: 4
+      mem_limit: 8m
+      mem_reservation: 5m`
+
+	content := []byte(ecsParamsString)
+
+	tmpfile, err := ioutil.TempFile("", "ecs-params")
+	assert.NoError(t, err, "Could not create ecs fields tempfile")
```

Comment:
Nit: I know this is not your fault since you were copying from pre-existing tests, but noting here that this should probably say "Could not create ecs params tempfile"

Reply:
done